docs/source/api.rst

```diff
@@ -37,6 +37,16 @@ Shortest Paths
    retworkx.num_shortest_paths_unweighted
    retworkx.unweighted_average_shortest_path_length
 
+.. _centrality:
+
+Centrality
```

Comment:
We can name this section something like `Graph Measures` and include here `retworkx.transitivity` and `retworkx.core_number` that are currently placed in the more general section `Other Algorithm Functions`.

Reply:
I intentionally followed the [networkx docs](https://networkx.org/documentation/networkx-1.10/reference/algorithms.html) in choosing this heading. My thinking was that, if retworkx were to develop anything like feature parity (a big job), then following their doc organization, which seems to be done very well, would be a good idea.

Reply:
Yeah, that's a good idea. It's just right now that retworkx lacks a ton of functionality comparing to networkx so having more general sections where we can place a few different functions seemed more appropriate. But it's totally fine to leave it as it is.